src/Page/Partners/Partner_.elm

```diff
@@ -193,6 +194,18 @@ viewAddress maybeAddress =
         Nothing ->
             p [ css [ contactItemStyle ] ] [ text (t PartnerAddressEmptyText) ]
 
+partnerLogo : Maybe String -> Html msg
+partnerLogo maybeLogoUrl =
+  case maybeLogoUrl of
+    Just logoUrl ->
+      div [ css [ partnerLogoContainer] ] [
+        img [ src logoUrl, css [ partnerLogoStyle ] ] []
```

Comment:
Sorry I missed this before - but we should put some alt text here. Either empty or partnerName + " logo"